src/material.cpp

```diff
@@ -36,8 +36,8 @@ material_type::material_type()
 {
     _ident = material_id( "null" );
     _name = "null";
-    _salvage_id = "null";
-    _salvage_multiplier = 1.0;
+    _salvaged_into = itype_id( "null" );
+    _repaired_with = itype_id( "" );
```

Comment:
> Why empty rather than "null"?

I kept the original value.

> Also, eventually materials should have a consistency check that verifies the existence of those items.

Agreed.